File: fixtures/RH011/lang/after/CalculatorTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

class CalculatorTest {

    @Test
    @SuppressWarnings("unchecked")
    void addsTwoNumbers() {
        int result = Calculator.add(2, 3);
        assertEquals(5, result);
    }

    @Test
    @SuppressWarnings("unchecked")
    void subtractsTwoNumbers() {
        int result = Calculator.subtract(5, 2);
        assertEquals(3, result);
    }
}

File: fixtures/RH011/lang/after/calculator_test.go
package calculator

import "testing"

func TestAdd(t *testing.T) {
	//nolint
	result := Add(2, 3)
	if result != 5 {
		t.Errorf("Add(2, 3) = %d; want 5", result)
	}
}

func TestSubtract(t *testing.T) {
	//nolint
	result := Subtract(5, 2)
	if result != 3 {
		t.Errorf("Subtract(5, 2) = %d; want 3", result)
	}
}
